src/components/guess.js

```diff
@@ -2,18 +2,29 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
 class Guess extends Component {
+
   render() {
     const guess = this.props.guess;
     if (guess.get('status') === "correct") {
       let person = this.props.person;
       let firstName = person.getIn(['0','first_name']);
       let lastName = person.getIn(['0','last_name']);
       let personId = person.getIn(['0','person_id']);
+      let stints = person.getIn(['0','stints']);
+
+      let stint_info = new Set(stints.map(s => this.stintToString(s)));
```

Comment:
Using a Set here was based on an edge case where `stintToString()` is not unique for stints beginning and ending in the same month.

I saw this in [Stefan Karpinski's profile](https://www.recurse.com/directory/228-stefan-karpinski) where he started two one-week residencies in the same month, July 2013: 
```sql
SELECT first_name, last_name, stint_type, start_date, end_date 
FROM people 
INNER JOIN stints 
ON stints.person_id = people.person_id 
AND people.person_id = 228 
ORDER BY start_date; 
```

If a stint begins and ends in the same month, the `dateToString()` method will only render a single date, `Jul '13` in Stefan's case. So although stint metadata should be unique, the result of `dateToString()` may not be, and I didn't want to clutter the interface with text that didn't add useful information.

In Stefan's case, his two stints would render differently because his first stint began and ended in July, while his second stint began in July and ended in August. But I thought it couldn't hurt to account for this in the view knowing the date rendering may not be unique.

Reply:
Ah cool, I missed that distinction between the stint and its string rendering - thank you for explaining!